koku/api/report/gcp/provider_map.py

```diff
@@ -61,8 +66,9 @@ def __init__(self, provider, report_type):
                         {"field": "project_name", "operation": "icontains", "composition_key": "project_filter"},
                         {"field": "project_id", "operation": "icontains", "composition_key": "project_filter"},
                     ],
+                    "instance_type": {"field": "instance_type", "operation": "icontains"},
                 },
-                "group_by_options": ["account", "region", "service", "project"],
+                "group_by_options": ["account", "region", "service", "project", "instance_type"],
```

Comment:
Similarly, are we okay without this as a group by option? It isn't technically something we support grouping on, the instance type API just inherently groups by it.